soh/src/overlays/actors/ovl_Bg_Jya_Lift/z_bg_jya_lift.c

```diff
@@ -96,7 +96,11 @@ void BgJyaLift_DelayMove(BgJyaLift* this, GlobalContext* globalCtx) {
     if (Flags_GetSwitch(globalCtx, this->dyna.actor.params & 0x3F) || (this->moveDelay > 0)) {
         this->moveDelay++;
         if (this->moveDelay >= 20) {
-            OnePointCutscene_Init(globalCtx, 3430, -99, &this->dyna.actor, MAIN_CAM);
+            // Skip this cutscene if using Sunlight Arrows in rando, since activating the switch while
+            // not standing on the platform will cause the cutscene to show the unloaded central room
```

Comment:
did you look into ways to make it so we don't need to skip this cutscene if we have sunlight arrows enabled but are still using the mirror shield to reflect the sunlight?

if not that's cool, but if you did and already know what some of the possible problems with trying to do that are it'd be good to have those documented somewhere (even if that somewhere is just a reply to this comment)

Reply:
I hadn't really looked into it, but I'm happy to investigate a little more.  I knew I didn't want to have a weird-looking unloaded room showing by accident, and skipping that cutscene was a pretty straightforward way to avoid that.

Reply:
Found a way to do this by looking at Link's coordinates, so this way the cutscene is only skipped if Link is not standing on the platform (regardless of whether that switch is hit by Light Arrows or by Mirror Shield reflection).

Footage: https://cdn.discordapp.com/attachments/988962246585634856/1016756446521733221/chainplatform.mp4